client/app/i18n/locales/de/vizlib.json

```
@@ -0,0 +1,261 @@
+{
+  "Cancel": "Abbrechen",
+  "Apply": "Anwenden",
+  "Something went wrong": "Etwas ist schief gelaufen",
+  "Align left": "Linksbündig",
+  "Align center": "Zentriert",
+  "Align right": "Rechtsbündig",
+  "Formatting Numbers": "Zahlen formatieren",
+  "Formatting Dates and Times": "Datum und Uhrzeit formatieren",
+  "Tick Formatting": "Tick-Formatierung",
+  "X Axis Label": "X-Achsenbeschriftung",
+  "Y Axis Label": "Y-Achsenbeschriftung",
+  "Boxplot (Deprecated)": "Boxplot (Veraltet)",
+  "Scale": "Skala",
```

not sure where this appears but if this means like a ratio, then "Maßstab" is better
```suggestion
  "Scale": "Maßstab",
```